jibri/rootfs/defaults/jibri.conf

```diff
@@ -20,6 +21,11 @@
 jibri {
     // A unique identifier for this Jibri
     id = "{{ .Env.JIBRI_INSTANCE_ID }}"
+    // Whether or not Jibri should return to idle state after handling
+    // (successfully or unsuccessfully) a request.  A value of 'true'
+    // here means that a Jibri will NOT return back to the IDLE state
+    // and will need to be restarted in order to be used again.
+    single-use-mode = {{ .Env.JIBRI_SINGLE_USE_MODE }}
```

Comment:
You are not using the variable you decalred here. Use $JIBRI_SINGLE_USE_MODE.

Reply:
You are so right. I corrected that line.